ignite3/src/main/java/site/ycsb/db/ignite3/IgniteClient.java

```diff
@@ -46,9 +47,16 @@ public Status insert(String table, String key, Map<String, ByteIterator> values)
       Tuple tValue = Tuple.create(fieldCount);
       values.forEach((field, value) -> tValue.set(field, value.toString()));
 
-      kvView.put(null, tKey, tValue);
+      wrapWithTx(() -> {
```

Comment:
Microbenchmarks should be as simple as possible. There should be no `if` clauses on hot execution paths. So I'd suggest to create separate classes for benchmarks which use explicit transactions. Keep existing benchmarks intact. Here it would me much better to "extend" the logic rather than "modify" it.

Reply:
Created draft with your suggestion on KV client.
https://github.com/gridgain/YCSB/pull/55

Reply:
@iartiukhov is correct and a separate benchmark should be created for explicit transactions.
Let's avoid creating new PR here because there is no alternative.